<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spaceForm</class>
 <widget class="QWidget" name="spaceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>251</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>445</width>
    <height>251</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>445</width>
    <height>251</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>421</width>
     <height>231</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>421</width>
     <height>231</height>
    </size>
   </property>
   <widget class="QLabel" name="lb_content">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pb_addFriend">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>加好友</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_jubao">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_icon">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>11</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_time">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>time</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>421</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_tui">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>踩</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_comment">
       <property name="text">
        <string>评论</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_good">
       <property name="text">
        <string>点赞</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>451</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 255, 225);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
